furusiya/model/entities/game_object.py

```diff
@@ -94,3 +95,9 @@ def draw(self):
     def clear(self):
         # erase the character that represents this object
         Game.ui.con.draw_str(self.x, self.y, ' ', self.color)
+
+    def die(self):
+        Game.area_map.entities.remove(self)
+        self.clear()
+        self.name = ''
+        self.blocks = False
```

Comment:
You know what would be cool? A map engulphed in flames and smothered in enemies. As the pillars of fire receed, you see dozens of red eyes peering at you from the darkness...